src/main/java/seedu/taassist/logic/commands/AssignCommand.java

```diff
@@ -62,8 +61,8 @@ public CommandResult execute(Model model) throws CommandException {
         List<Student> lastShownList = model.getFilteredStudentList();
         List<Student> studentsToAssign;
         try {
-            studentsToAssign = ParserStudentIndexUtil.parseStudentsFromIndices(indices, lastShownList);
-        } catch (ParseException pe) {
+            studentsToAssign = IndexUtil.getAtIndices(lastShownList, indices);
+        } catch (IndexOutOfBoundsException pe) {
```

Comment:
```suggestion
        } catch (IndexOutOfBoundsException ioobe) {
```

Perhaps consider changing the exception name to match the exception type? This is a very minor issue though.